AEXS/aex-2.md

```diff
@@ -23,28 +23,17 @@ By defining the standard way of communication between SDK(aepps) and Wallet we w
 
 ## Specification
 
-### JSON-RPC 2.0 Methods
-
-#### General
-
-- `ping/pong`: general ping/pong messages to check liveness. Implementation of this method is not mandatory for wallets communicating over a transport layer that have native support for liveness check.
-
-    ##### Parameters
+### Naming Convention
 
-    `Object`
-
-  - `id` - A unique identifier, must conform to the [UUID v4 standards](https://tools.ietf.org/html/rfc4122#page-14)
+This document tries to closely follow `who.what.how` naming convention for JSON-RPC methods(wherever possible).
```

Comment:
Do you have any explanation/motivation/reasoning of this naming convention? Maybe we can try to make it a separate standard to use it in all AEXs?

Reply:
as mentioned earlier on several occasions we follow the naming convention as per the naming convention used earlier in aeternity. you can find more details here https://github.com/aeternity/protocol/tree/master/channels#terms

Reply:
Also, there are a missed space before "("